src/main/java/picard/analysis/directed/InsertSizeMetricsCollector.java

```diff
@@ -168,6 +169,7 @@ public void addMetricsToFile(final MetricsFile<InsertSizeMetrics,Integer> file)
                             if (percentCovered >= 0.7 && metrics.WIDTH_OF_70_PERCENT == 0) metrics.WIDTH_OF_70_PERCENT = distance;
                             if (percentCovered >= 0.8 && metrics.WIDTH_OF_80_PERCENT == 0) metrics.WIDTH_OF_80_PERCENT = distance;
                             if (percentCovered >= 0.9 && metrics.WIDTH_OF_90_PERCENT == 0) metrics.WIDTH_OF_90_PERCENT = distance;
+                            if (percentCovered >= 0.9 && metrics.WIDTH_OF_95_PERCENT == 0) metrics.WIDTH_OF_95_PERCENT = distance;
```

Comment:
@nh13 shouldn't this be  percentCovered >= 0.95 ?

Reply:
<swearing ensues>